templatify.ts

```diff
@@ -23,7 +22,7 @@ function getNetworkNameForSubgraph(): string | null {
     case 'TokenUnion/polymarket-mumbai':
       return 'mumbai';
     default:
-      return 'matic';
+      return null;
   }
```

Comment:
Don't return any default to be more explicit. Also default in here makes lines 38-42 never being accessed.